pkg/storage/vulnerability.go

```diff
@@ -1264,7 +1265,8 @@ func (r *VulnerRepo) ListWebVulnerabilitiesByServiceID(ctx context.Context, serv
 				Reference:       dbVuln.Reference.String,
 				CreatedAt:       &dbVuln.CreatedAt.Time,
 				UpdatedAt:       &dbVuln.UpdatedAt.Time,
-				Instances:       []domain.WebVulnerabilityInstance{},
+				Title:           dbVuln.Title,
+				Severity:        dbVuln.Severity,
```

Comment:
`[Nit]` Even though behavior is the same (GoLang code-wise), I feel it's better to **be explicit on our code** that we're initializing Instances as empty here. (the zero-value of a slice is nil)


```suggestion
				Title:           dbVuln.Title,
				Severity:        dbVuln.Severity,
				Instances: []domain.WebVulnerabilityInstance{}
```

Or...

```suggestion
Title: dbVuln.Title,
Severity: dbVuln.Severity,
Instances: nil
```

Reply:
Thanks for this correction,